app/views/ducr_part_details.scala.html

```diff
@@ -65,6 +69,13 @@
           inputClasses = "govuk-input--width-3"
         )
 
+        @govukDetails(Details(
+            summary = Text(messages("ducrPartDetails.details.title")),
+            content = HtmlContent(messages("ducrPartDetails.details.text", link(
+                message = Html(messages("ducrPartDetails.details.text.link")),
+                href = Call("GET", config.ucrInformationUrl))))
```

Comment:
If I'm not wrong, the link should open in a new tab. In that case, should not the `target` be specified?